src/components/BookPage/AddReviewForm/index.js

```diff
@@ -11,6 +11,11 @@ function Form(props) {
   const [reviewCharCount, setReviewCharCount] = useState(0);
   const maxReviewLength = 450;
 
+  function handleInputChange(event, setState) {
+    const newValue = event.target.value;
+    setState(newValue);
+  }
```

Comment:
This is a nice approach. Please do this on the AddBook page as well for consistency